lib/configuration/variables/sources/instance-dependent/get-sls.js

```diff
@@ -27,6 +27,12 @@ module.exports = (serverlessInstance) => {
           if (!stage) stage = 'dev';
           return { value: stage };
         }
+        case 'region': {
```

Comment:
Region is internally considered as AWS specific. It's also the reason why it was specified to be put at `aws:` source

Reply:
While it's considered AWS specific, other major providers also have such concept and support it on provider level, however, I agree that it's tricky because it defaults to `us-east-1` which is AWS-specific. 

Azure: https://github.com/serverless/serverless-azure-functions/blob/c53f535204c467339e8ae1f6edb985c3280bc32c/src/models/serverless.ts#L39
Google: https://github.com/serverless/serverless-google-cloudfunctions/blob/master/provider/googleProvider.js#L107

Given above, do you think we should stick to `region` being in `aws` source?

Reply:
> Given above, do you think we should stick to region being in aws source?

I believe so. Main reason is that different dictionaries are used in different providers. While with `stage` we rely on same dictionary no matter what provider is used (`dev` serves as default for all)

Also provider may introduce their variable source as e.g. `azure:region`. So we do not block anything here. We're putting it on `aws` to actually not collide with how region is handled in case of other providers.